sbom/main.ts

```diff
@@ -0,0 +1,131 @@
+import { PhylumApi } from 'phylum';
+import { parse } from "https://deno.land/std@0.156.0/flags/mod.ts";
+
+const spdxVersion = "SPDX-2.2";
+const spdxId = "SPDXRef-DOCUMENT";
+const creator = "Tool: github.com/phylum-dev/cli";
+const dataLicense = "CC0-1.0"
+
+interface Dependency {
+    name: string;
+    version: string;
+}
+
+interface Package {
+    name: string;
+    version: string;
+    license: string;
+    type: string;
+    dependencies: Dependency[];
+}
+
+/**
+ * Build the homepage URL for the provided package.
+ */
+function getHomepage(pkg: Package) {
+    switch(pkg.type) {
+        case 'npm':
+            return `https://www.npmjs.com/package/${pkg.name}/v/${pkg.version}`;
+        case 'pypi':
+            return `https://pypi.org/project/${pkg.version}/${pkg.version}/`;
+        case 'rubygems':
+            return `https://rubygems.org/gems/${pkg.name}/versions/${pkg.version}`;
+        case 'maven':
+            return ``; // TODO: How do we handle this? Maven central? What about others?
```

Comment:
Should this just be `NOASSERTION` until we figure this out?

Reply:
Good catch, I think we _can_ build out the URL. I think I just wasn't aware of how to do it when I first wrote this piece up. I'll look into the implementation, and short of that will `NOASSERTION` it.

Reply:
Updated to build out the maven URLs.